projects/ng-components/lib/toggle-selector/toggle-selector.component.spec.ts

```diff
@@ -0,0 +1,172 @@
+        });
+
+        it("Should have two checked when both options are clicked", () => {
+            const element: HTMLElement[] = fixture.nativeElement.querySelectorAll(".toggle-selector");
+            element[1].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            const inputEl: DebugElement[] = fixture.debugElement.queryAll(By.css("input"));
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeFalsy();
+
+            element[0].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeTruthy();
+        });
+
+        it("should uncheck when previously checked option is clicked", () => {
+            const element: HTMLElement[] = fixture.nativeElement.querySelectorAll(".toggle-selector");
+            element[1].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            const inputEl: DebugElement[] = fixture.debugElement.queryAll(By.css("input"));
+
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeFalsy();
+
+            element[1].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            expect(inputEl.some(el => el.nativeElement.checked === true)).toBeFalsy();
+        });
+    });
+
+    describe("Things that should happen when multi is false", () => {
+        beforeEach(() => {
+            component.multi = false;
+            component.list = [
+                { value: "1", label: "Four mississipi" },
+                { value: "2", label: "Five mississipi" },
+            ];
+            fixture.detectChanges();
+        });
+
+        it("Should have type radio when multi is false", () => {
+            const elements: DebugElement[] = fixture.debugElement.queryAll(By.css(".toggle-selector input[type=radio]"));
+            expect(elements.length).toBe(component.list.length);
+        });
+
+        it("should not have anything checked", () => {
+            const inputEl: DebugElement[] = fixture.debugElement.queryAll(By.css("input"));
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeFalsy();
+        });
+
+        it("Should have only one checked when an option is clicked and move to another option if it's clicked", () => {
+            const element: HTMLElement[] = fixture.nativeElement.querySelectorAll(".toggle-selector");
+            element[1].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            const inputEl: DebugElement[] = fixture.debugElement.queryAll(By.css("input"));
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeFalsy();
+            expect(inputEl[1].nativeElement.checked).toBeTruthy();
+
+            element[0].dispatchEvent(new Event("click"));
+            fixture.detectChanges();
+            expect(inputEl.every(el => el.nativeElement.checked === true)).toBeFalsy();
```

Comment:
Same here, just return `checked`
```suggestion
            expect(inputEl.every(el => el.nativeElement.checked)).toBeFalsy();
```